proposed/0000-Farm-attempt.md

```diff
@@ -0,0 +1,56 @@
+- Feature Name:Farm attempt 
+- Type new feature
+- Related components maidsafe_vault
+- Start Date: 24-06-2015 
+- RFC PR:
+- Issue number: 
+
+# Summary
+
+This proposal outlines an initial design to test the efficiency and security of farming attempts.  This involves making use of the Farming Rate (FR) which is the rate at which farming attempts are made.  It is assumed the farming rate will include payments to care development, content producers and most significantly farmers (or providers of the network resource). 
+
+# Motivation
+
+To ensure network stability and resource availability a reward mechanism is required to ensure the use of resource is balanced with the supply of resource. 
+
+# Scope of work
+
+This implementation will be solely in the vault library. This will include several steps. The client lib will require to be aware of the actual safecoin type (again `StructuredData`). 
+
+# Detailed design
+
+ON receipt of a `Get` request for `ImmutableData` the `DataManager` calculates the SHA512 Hash of the `name()` of the `ImmutableData` concatenated with the
+`PmidHolder`. This result is then modulo divided by the farming rate (FR). The rate attempts should divide the award between 4 distinct groups. 
```

Comment:
I assume `PmidHolder` means one of the PmidNode. 
Currently DataManager Sends on the get request to all the pmid_nodes. 
So, we need to have a deterministic way to get the PmidHolder(s), so that the hash at different group nodes yield same result.